src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,168 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev1 (02/24/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade to Pex 2.1.4 (#8881)
+  `PR #8881 <https://github.com/pantsbuild/pants/pull/8881>`_
+
+* Register `goal2` as `goal` if in v2-exclusive mode. (#9164)
+  `PR #9164 <https://github.com/pantsbuild/pants/pull/9164>`_
+
+* Turn on `--v2` by default now that the V2 backends are not loaded by default (#9007)
+  `PR #9007 <https://github.com/pantsbuild/pants/pull/9007>`_
+
+* Reapply "Switch to dot idea project type" PR (#9105)
+  `PR #9105 <https://github.com/pantsbuild/pants/pull/9105>`_
```

Comment:
@wisechengyi originally this was marked API change, so I kept it here.

Reply:
sounds good. thanks!